src/transport.cpp

```diff
@@ -3,6 +3,10 @@
 #if not defined(PLATFORM_ESP)
   #include <transport_picoquic.h>
 #endif
+#include <memory>
+#include <stdexcept>
+#include <spdlog/logger.h>
```

Comment:
It preferred this over
```cpp
#include <spdlog/spdlog.h>
```
?

Reply:
Yeah, it doesn't know about umbrella headers that don't actually define the thing directly, although we can declare them in a mapping file so it does know. Here it didn't seem too horrendous so I didn't bother. But that's the same thing that I'm raising in my comment about `<transport/*.h>`. We can add a pragma in that case if we want.